.github/workflows/tests-contracts.yaml

```diff
@@ -0,0 +1,69 @@
+name: tests-contracts
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  TEST_DOCKER_IMAGE: finschia/finschianode
+  GO_VERSION: "1.20"
+
+jobs:
+  tests-contracts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: Finschia/finschia
+      - uses: docker/setup-buildx-action@v3
+
+      - name: Docker pull
+        run: |
+          docker pull finschia/finschianode:2.0.1
+          docker image ls -a
+      - name: Set configure file
+        run: |
+          export FNSAD="docker run -i --rm -p 26656:26656 -p 26657:26657 -v ${pwd}/.finschia:/root/.finschia ${{ env.TEST_DOCKER_IMAGE }} fnsad"
+          ./init_single.sh
+        shell: bash
+
+      - name: Start docker container
+        id: docker_start
+        run: |
+          container_id=$(docker run -d \
+            -p 26656:26656 -p 26657:26657 \
+            -v ${pwd}/.finschia:/root/.finschia \
+            ${{ env.TEST_DOCKER_IMAGE }} fnsad start)
+          echo "container_id=$container_id" >> "$GITHUB_OUTPUT"
+          sleep 5
```

Comment:
Is this the same as https://github.com/Finschia/finschia-wasm/blob/main/scripts/start_integration_test_contracts.sh　